client/driver/driver.go

```diff
@@ -133,7 +133,6 @@ func TaskEnvironmentVariables(ctx *ExecContext, task *structs.Task) environment.
 		if len(task.Resources.Networks) > 0 {
 			network := task.Resources.Networks[0]
 			env.SetTaskIp(network.IP)
```

Comment:
This has to be done inside the drivers though since port forwarding is a driver specific capability.